charts/quantum-serverless/charts/gateway/values.yaml

```diff
@@ -3,44 +3,60 @@
 # Declare variables to be passed into your templates.
 
 replicaCount: 1
+nameOverride: "gateway"
+fullnameOverride: "gateway"
+
+# set true when cert-manager is available in the cluster
 useCertManager: false
 
 application:
-#  command: [ "gunicorn", "gateway.wsgi:application", "--bind", "0.0.0.0:8000", "--workers=3" ]
+  # enable debug mode
   debug: 0
-  siteHost: "http://127.0.0.1:8000"
-  rayHost: "http://ray:8265/"
+
+  # gateway site url - http://<fullnameOverride>:<service.port>
+  siteHost: "http://gateway:8000"
+  # authentification setting
   auth:
     token: {}
     keycloak: {}
-  superuser:
-    enable: true
+  # django secret key - any secret string
+  django:
```

Comment:
This key is not needed now, it goes in the secrets part.

Reply:
I see. I take it out.